.goreleaser.yml

```diff
@@ -0,0 +1,51 @@
+builds:
+  - 
+    id: 'manager'
+    main: ./cmd/manager/main.go
+    binary: addon-manager
```

Comment:
The binary that is generated by make is usually called just `manager`, and this is referenced in the Dockerfile, I think we need to change your ref here or in the build and Dockerfile.

Reply:
Hi Kevin, I have already updated the manager binary

Reply:
I am still facing this issue. Technically, the github token still missing when calling goreleaser

<img width="903" alt="Screen Shot 2019-09-13 at 10 59 12 AM" src="https://user-images.githubusercontent.com/4967217/64884066-a3aa1200-d615-11e9-842b-25f1f96e741f.png">